docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -79,6 +79,53 @@ Hopefully this change won't affect your code but will allow for more customizati
   The TextAreaField component does apply 1 style prop directly on the `textarea` element: `resize`. We felt this one makes sense to apply direclty on the the `textarea` element and not the wrapper element.
 </Alert>
 
+#### Automatic signin on signup logic moved to `aws-amplify`. 
```

Comment:
Nit: can we add '5.' to the start of this header to match the previous headers?

Reply:
On a previous code review it was asked we split this out separately from the other steps, to make this self contained.  I think this looks fine

Reply:
Talked offline about this, fixed